examples/viewer.py

```diff
@@ -723,7 +723,7 @@ def mouse_press_event(self, event: Application.MouseEvent) -> None:
                 hit_object, ao_link = -1, -1
                 hit_info = raycast_results.hits[0]
 
-                if hit_info.object_id >= 0:
+                if hit_info.object_id > habitat_sim.stage_id:
```

Comment:
Using "greater than" here still assumes that stage_id will always be less than all object ids. While this is the case currently does it defeat the purpose of having a variable?